theme/tutorial.less

```diff
@@ -696,6 +696,11 @@ code.lang-filterblocks {
 
 /* > Small Monitor */
 @media only screen and (min-width: @computerBreakpoint) {
+    .shrink {
+        .landscape.only {
```

Comment:
I think I'd personally prefer the button to always go small in those dimension rather than only when strictly necessary -- it's very cramped even when everything shows up on screens like that and shrinking the button actually leaves a nice amount of room for everything to settle in. Maybe a quick topic for a design office hour?

Reply:
oh yeah sure! perfect timing